refinery/data_set_manager/tests.py

```diff
@@ -1,48 +1,43 @@
 from StringIO import StringIO
 import contextlib
-import json
 import logging
 import os
 import re
 import shutil
 import tempfile
-from urlparse import urljoin
 import uuid
 
-from django.conf import settings
 from django.contrib.auth.models import User
 from django.core.files.uploadedfile import (InMemoryUploadedFile,
                                             SimpleUploadedFile)
 from django.core.management import call_command, CommandError
 from django.db.models import Q
 from django.http import QueryDict
-from django.test import LiveServerTestCase, TestCase, override_settings
+from django.test import TestCase, override_settings
 
 from celery.states import FAILURE, PENDING, STARTED, SUCCESS
 from djcelery.models import TaskMeta
 from factory_boy.utils import (create_dataset_with_necessary_models,
                                make_analyses_with_single_dataset)
-from guardian.shortcuts import assign_perm
 from haystack.exceptions import SkipDocument
 import mock
-from mock import ANY
 from override_storage import override_storage
-from rest_framework.test import APIClient, APIRequestFactory, APITestCase
+from rest_framework.test import APITestCase
 
 import constants
 from core.models import (INPUT_CONNECTION, OUTPUT_CONNECTION, Analysis,
                          AnalysisNodeConnection, DataSet, InvestigationLink)
 from core.tests import TestMigrations
-import data_set_manager
```

Comment:
This was probably introduce to resolve a circular import? I manual tested and didn't have any conflicts. @scottx611x